### PR TITLE
Create eram.yml

### PR DESCRIPTION
**Pull request by Zero3K**

### CODE REVIEW

File: _data/projects/eram.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- ram
+- disk
+- cache
+- storage
+- temp
+- files
+- c
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/username/project/labels/up-for-grabs
```
Comment (ritwik12) [inline _data/projects/eram.yml]: @Zero3K Please correct this to https://github.com/Zero3K/ERAM/labels/up-for-grabs

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/eram.yml (hunk 1 above)
Comment (Zero3K) [reply _data/projects/eram.yml]: Okay. Its been done.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@Zero3K Thanks!!

**Comment by Zero3K** [comment]
You're welcome.

**Pull request merged**